tests/Feature/ResponsiveImages/ResponsiveImageGeneratorTest.php

```diff
@@ -29,4 +31,17 @@ public function its_conversions_can_have_responsive_images()
 
         $this->assertFileExists($this->getTempDirectory('media/1/responsive-images/test___thumb_50_41.jpg'));
     }
+
+    /** @test */
+    public function it_triggers_an_event_when_the_responsive_images_are_generated()
+    {
+        Event::fake([ResponsiveImagesGenerated::class]);
```

Comment:
Would the test work with only `Event::fake()`?

Reply:
I prefer to keep tests as specific as possible to avoid unforeseen future side-effects. But I've removed the `[]` because it was unnecessary.